### ISSUE TITLE
Page showing register option even after login

### ISSUE THREAD

**Issue by fork52** [CodeTiger927/StonksCF#2]
After logging in, the page still shows a register option which can be confusing for users. Should be hidden once you login/register.

**Comment by sanyam-git** [comment]
Also page is showing the Login option. Both options Login and Register should be replaced with a Log Out kind of button.

@CodeTiger927 super cool product !

**Comment by fork52** [comment]
Agree with @sanyam-git. Amazing work @CodeTiger927 !

**Comment by CodeTiger927** [comment]
Yep, will definitely work on this as soon as I get a chance/finalize next update
